doc/tutorials/rt_industry.rst

```diff
@@ -0,0 +1,197 @@
+   |                      |                  +----------------------+-----------------------------------------------------------+
+   |                      |                  | System memory        | - 8GiB SODIMM DDR4 2400 MHz                               |
+   |                      |                  +----------------------+-----------------------------------------------------------+
+   |                      |                  | Storage capabilities | - SATA: 1TB WDC WD10SPZX-22Z                              |
+   |                      |                  |                      | - NVMe: 256G Intel Corporation SSD Pro 7600p/760p/E 6100p |
+   +----------------------+------------------+----------------------+-----------------------------------------------------------+
+
+Software Setup
+==============
+
+.. table:: Software Setup
+   :widths: auto
+   :name: Software Setup
+
+   +---------------------+------------------------+------------------------------------------+------------------------------------------------------------------------------------------------+
+   | Clear Linux Version | ACRN tag (hypervisor)  | Description                              | Download Link                                                                                  |
+   +=====================+========================+==========================================+================================================================================================+
+   | 29750               |                        | **Host machine:**                        | `link <https://download.clearlinux.org/releases/29750/clear/clear-29750-live-server.iso.xz>`__ |
+   |                     |                        | build ACRN hypervisor, tools and kernels |                                                                                                |
+   +---------------------+------------------------+------------------------------------------+------------------------------------------------------------------------------------------------+
+   | 30690               | v1.2                   | **KBL NUC:**                             | `link <https://download.clearlinux.org/releases/30690/clear/clear-30690-live-server.iso.xz>`__ |
+   |                     |                        | set up Service VM or Native OS           |                                                                                                |
+   +---------------------+------------------------+------------------------------------------+------------------------------------------------------------------------------------------------+
+
+Install build tools and dependencies
+====================================
+On the Clear Linux build machine, install the necessary tools in order to build ACRN
+hypervisor, Service VM and Preempt-RT kernels::
+
+  $ sudo swupd bundle-add os-clr-on-clr os-core-dev python3-basic
+  $ pip3 install --user kconfiglib
```

Comment:
You should remind folks to verify ``~/.local/bin`` is on their PATH when using ``--user``

Reply:
Add one more step before using ``pip3 install --user``:
``echo "export PATH=\$HOME/.local/bin:\$PATH" >> ~/.bashrc && source ~/.bashrc``